src/drivers/general/queries/ExtendedDeprecatedApis/OpaqueMdlUse.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+	<overview>
+		<p>
+			Deprecated, insecure APIs should not be used.
+		</p>
+	</overview>
+	<recommendation>
+		<p>
+			Instead of using deprecated APIs, driver developers should refer to the output of the rule and replace any calls to deprecated APIs with the recommended replacements.  These recommendations are based on the MSDN articles for Code Analysis rules C28719, C28726, and C28750, as linked in the references section below.
```

Comment:
I'd add a statement about what to do if there isn't a recommended replacement since so many of them didn't have one listed.

Reply:
Since ultimately the correct response is going to depend on the API the customer is using, there's not too much specific I can say here, but I'll add a note that if there is no listed replacement the developer should investigate potential replacement functions (this is also what gets printed by the rule itself when run.)